src/client/Client.cc

```diff
@@ -3643,6 +3643,7 @@ void Client::put_cap_ref(Inode *in, int cap)
       }
     }
     if (last & CEPH_CAP_FILE_CACHE) {
+      signal_context_list(in->waitfor_caps);
```

Comment:
if we want to process the `waitfor_caps` context list, how about `Client::signal_caps_inode`?